### PR TITLE
TLS 1.3:Add multiple pre-configured psk test for server

### LINKED ISSUE #6053: TLS 1.3: Add support to handle list of PSKs on server side
Follow-up of #5564 to support on server side the handling of a list of possible PSKs.

Testing: see #6052.

### PR DESCRIPTION
**Pull request by yuhaoth**
## Description

Fix #6053 

It has been supported, but there are no tests for it . 

## Status
**READY**

### CODE REVIEW

File: tests/opt-testcases/tls13-kex-modes.sh [hunk 1]
```diff
@@ -17,6 +17,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+P_SRV_PSK_LIST="psk_list=abc,dead,def,beef,Client_identity,6162636465666768696a6b6c6d6e6f70"
```
Comment (ronald-cron-arm) [inline tests/opt-testcases/tls13-kex-modes.sh:20]: To exercise in different ways the PSK look-up code on server side, what about to have 3 lists:
1) One with the Client_identity PSK being first in the list
2) One with the Client_identity PSK being last in the list (as the current one)
3) One with the Client_identity PSK being in the middle
and to use them in a randomly manner in the various tests.

**Review by ronald-cron-arm — CHANGES_REQUESTED** [review]
Just one suggestion.

**Review by ronald-cron-arm — APPROVED** [review]
LGTM

**Comment by ronald-cron-arm** [comment]
@xkqian could you please review this PR. This looks good to me.

File: tests/opt-testcases/tls13-kex-modes.sh [hunk 2]
```diff
@@ -18,6 +18,15 @@
 # limitations under the License.
 #
 
+get_srv_psk_list ()
+{
+    case $(( TESTS % 3 )) in
```
Comment (xkqian) [inline tests/opt-testcases/tls13-kex-modes.sh:23]: What's the value of TESTS?

Comment (yuhaoth) [reply tests/opt-testcases/tls13-kex-modes.sh:23]: It is set by `run_test` function.

**Review by xkqian — APPROVED** [review]
LGTM

**Pull request merged**